.github/workflows/develop_pa-test-internal-testserver-app.yml

```diff
@@ -49,6 +49,8 @@ jobs:
         registry: https://powerauthextendedtest.azurecr.io/
         username: ${{ secrets.AZUREAPPSERVICE_CONTAINERUSERNAME }}
         password: ${{ secrets.AZUREAPPSERVICE_CONTAINERPASSWORD }}
+    - name: Copy liquibase files
+      run: powerauth-test-server/copy_liquibase.sh
```

Comment:
The script is usually expected to be executed in the context of directory he resides in. I suggest to do
```
run: |
        cd powerauth-test-server
       ./copy_liquibase.sh
```
and leave the copy_liquibase.sh script unchanged.

Reply:
Thanks for the tip!